automatic/dropbox/update.ps1

```diff
@@ -14,26 +14,56 @@ function global:au_SearchReplace {
 }
 
 function global:au_GetLatest {
-    $downloadEndpointUrl = 'https://www.dropbox.com/download?full=1&plat=win'
-    $versionRegEx = '.*Dropbox%20([0-9\.]+).*'
-    $downloadUrl = Get-RedirectedUrl $downloadEndpointUrl
-    $version = $downloadUrl -replace $versionRegEx, '$1'
+    $releases = 'https://www.dropboxforum.com/t5/Desktop-client-builds/bd-p/101003016'
+    $HTML = Invoke-WebRequest -UseBasicParsing -Uri $releases
+    $stable_builds = @()
+    $beta_builds = @()
+    $HTML.Links | foreach {
+  	  if ($_.href -match "stable" ) { $stable_builds += $_.href }
+  	  if ($_.href -match "beta" ) { $beta_builds += $_.href }
```

Comment:
@AdmiringWorm 
I presume you are referring to this but of code? You want me to rename it to like nonstable. So as to not refer to beta? 
This I can work on later, but without it the whole check is pointless.